fsd_config/form_jsons/cyp_r1/applicant-information-cyp.json

```diff
@@ -545,7 +545,16 @@
     }
   ],
   "fees": [],
-  "outputs": [],
+  "outputs": [
+    {
+      "name": "update-form",
+      "title": "Update form in application store",
```

Comment:
`"title": "Update form in application store",` is this displayed on frontend ?

Reply:
No, it would be displayed on the designer for out reference only. User would never see it